src/stores/Binding/Store.ts

```diff
@@ -139,6 +142,13 @@ const hydrateSpecificationDependentState = async (
         );
     }
 
+    const trialCollections = await evaluateTrialCollections(
+        Object.keys(get().bindings),
+        getTrialOnlyPrefixes
+    );
+
+    get().setCollectionMetadata(trialCollections);
```

Comment:
I don't think this belongs in here as it is not relying on the draft spec at all

Reply:
It is hydrating binding dependent state.